plugins/dependencies/src/main/kotlin/dependencies/Testing.kt

```diff
@@ -88,8 +88,13 @@ object Testing {
             const val piTest = "$artifactPrefix-pitest:_"
         }
 
+        @Suppress("DEPRECATION")
         val extensions = Extensions
 
+        @Deprecated(
+            message = "Since Kotest 4.5.0 extensions have a separate lifecycle per extension",
+            replaceWith = ReplaceWith("Testing.KotestExtensions")
```

Comment:
Testing.kotestExtensions

Reply:
I made added `@Deprecated` on the property with `replaceWith = ReplaceWith("Testing.kotestExtensions")` (e.g. replace with new property), but left the object deprecation with `replaceWith ...` refering to the new object. Does that make sense, or should everything refer to the new property?

Reply:
Yes that maake sense